cost-analyzer/values.yaml

```diff
@@ -607,7 +607,11 @@ kubecostModel:
       # - datadog
 
     # pre-existing secret for plugin configuration
-    configSecret: kubecost-plugin-secret
+    existingCustomSecret:
+      enabled: false
+      name: ""  # name of the secret containing plugin config
+
+    secretName: kubecost-plugin-secret
```

Comment:
Do you think we need to build any backwards compatibility here? Since we are changing value `configSecret` to `secretName`?

Reply:
I'm not sure. I figured this feature is new enough that not many are likely using it yet.

Reply:
I'm ok with that! Are @chipzoller and @kwombach12 good with that?

Reply:
I also don't have a sense of how many are using this capability. Will wait for Kai.

Reply:
I really don't think we have much usage here yet, and given external costs (datadog) is considered 'beta', I am ok to move forward without backwards compatibility so long as we call out this change in release notes. Does that seem reasonable?

Reply:
Yep seams reasonable. I'm good to merge then!